cogboard-webapp/package.json

```diff
@@ -3,29 +3,29 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@emotion/core": "^10.0.14",
-    "@emotion/styled": "^10.0.14",
-    "@material-ui/core": "^4.2.1",
-    "@material-ui/icons": "^4.2.1",
-    "@reach/router": "^1.2.1",
-    "copy-to-clipboard": "^3.2.0",
+    "@emotion/core": "^10.0.28",
+    "@emotion/styled": "^10.0.27",
+    "@material-ui/core": "^4.9.11",
+    "@material-ui/icons": "^4.9.1",
+    "@reach/router": "^1.3.3",
+    "copy-to-clipboard": "^3.3.1",
     "http-proxy-middleware": "^0.20.0",
     "js-cookie": "^2.2.1",
     "jwt-decode": "^2.2.0",
-    "moment-timezone": "^0.5.26",
+    "moment-timezone": "^0.5.28",
     "ramda": "^0.26.1",
-    "react": "^16.8.6",
-    "react-dnd": "^9.3.4",
-    "react-dnd-html5-backend": "^9.3.4",
-    "react-dom": "^16.8.6",
+    "react": "^16.13.1",
```

Comment:
revert or check licensing of changed libraries if they meet Apache 2 license

Reply:
Everything is still on MIT license.

Reply:
licenses unchanged

Reply:
nice !